lusidtools/cocoon/cocoon.py

```diff
@@ -32,10 +32,19 @@ def load_instrument_batch(
         """
         Upserts a batch of instruments to LUSID
 
-        :param lusid.utilities.ApiClientFactory api_factory: The api factory to use
-        :param list[lusid.models.InstrumentDefinition] instrument_batch: The batch of instruments to upsert
-
-        :return: UpsertInstrumentsResponse: The response from LUSID
+        Parameters
+        ----------
+        api_factory : lusid.utilities.ApiClientFactory
+            The api factory to use
+        instrument_batch : list[lusid.models.InstrumentDefinition]
+            The batch of instruments to upsert
+        **kwargs
+            arguments specific to each call e.g. effective_at for holdings, unique_identifiers
```

Comment:
Can we please put the parameters themselves in other parameters and remove kwargs from here?

Reply:
Can we do this everywhere kwargs is found please?

Reply:
created jira 
https://finbourne.atlassian.net/browse/DEV-7497